### DISCUSSION TITLE
解构字段相关用法

### DISCUSSION THREAD

**Discussion by Nokecy** [alibaba/formily#1701] [Q&A 2.x] [answered]
https://codesandbox.io/s/exciting-margulis-gy9ey?file=/App.tsx

使用{label:A,value:B} 这样的解构用法，默认值不是undefined 而是{} ，这样placeholder 就没有显示出来

**Comment by janryWang** [comment]
没看懂你的例子

**Comment by liuweiGL** [comment]
感觉这个问题应该是 ui 侧来解决的，因为你的组件既然选择接收 object 类型的 value，就得提供对应的 isEmpty 逻辑。

**Comment by liuweiGL** [comment, marked as answer]
你可以把 Select 封装一层吧，加个 object 判空操作。
